src/radical/pilot/agent/scheduler/base.py

```diff
@@ -450,11 +453,14 @@ def _change_slot_states(self, slots, new_state):
     #
     # NOTE: any scheduler implementation which uses a different nodelist
     #       structure MUST overload this method.
-    def slot_status(self, msg=None):
+    def slot_status(self, msg=None, uid=None):
         '''
         Returns a multi-line string corresponding to the status of the node list
         '''
 
+        if not _debug:
+            return
+
```

Comment:
Yeah, we should use something like `debug_10` or so.  Let me try if I get that to work.  For now though this is rather useful :-)

Reply:
Done